applications/nrf_desktop/doc/wheel.rst

```diff
@@ -1,7 +1,56 @@
 .. _wheel:
 
-Wheel module
+wheel module
 ############
 
-Wheel module is responsible for generating events related to the rotation of
+The ``wheel`` module is responsible for generating events related to the rotation of
 the mouse wheel.
+
+Module Events
+*************
+
++----------------+-------------+--------------+-----------------+------------------+
+| Source Module  | Input Event | This Module  | Output Event    | Sink Module      |
++================+=============+==============+=================+==================+
+|                |             | :ref:`wheel` | ``wheel_event`` | :ref:`hid_state` |
++----------------+-------------+--------------+-----------------+------------------+
+
+Configuration
+*************
+
+The module is enabled by the ``CONFIG_DESKTOP_WHEEL_ENABLE`` define.
+
+For detecting rotation, ``wheel`` uses Zephyr's QDEC driver. The module can be enabled
+only when QDEC is configured in DTS (:option:`CONFIG_HAS_DTS_QDEC` is set).
+
+The QDEC DTS configuration specifies how many steps are done during one full angle
+The sensor reports the rotation data in angle degrees.
+``wheel`` configuration option
```

Comment:
There is something wrong with lines 28-31. I guess lines 28 and 29 can be removed, and lines 30-31 are their new version.

Reply:
Thanks. I missed it. This is the merging issue.